WARNING: Calling get_html_theme_path is deprecated. If you are calling it to define html_theme_path, you are safe to remove that code.
[...]
build finished with problems, 1 warning (with warnings treated as errors).

https://github.com/OISF/suricata/pull/13742

Index: doc/userguide/conf.py
--- doc/userguide/conf.py.orig
+++ doc/userguide/conf.py
@@ -138,7 +138,6 @@ if not on_rtd:
     try:
         import sphinx_rtd_theme
         html_theme = 'sphinx_rtd_theme'
-        html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
     except:
         html_theme = 'default'
 else:
